eBookReader/chat.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Q&A</title>
+	<link rel="stylesheet" type="text/css" href="styleIndex.css"/>
+	<link rel="stylesheet" type="text/css" href="debugStyle.css"/>
+</head>
+<body>
+<header>
+	<p class = "name">John Doe</p>
+	<p class = "userStats">
+		Contribution points: 91
+		<a href="#" class="iconButton">?</a>
+	</p>
+</header>
+
+<h2>According to the author, why do mass extinctions happen?</h2>
+
+<div class = "chatPanel">
+	<div class = "chatMessage">I believe that on page 3, the author clearly states that extinctions happen for three reasons.</div>
```

Comment:
This would probably be better (i.e. more semantic) as a p element rather than a div.